src/Backend/WASMWrapper.js

```diff
@@ -1,17 +1,19 @@
 import DataSeriesFactory from './DataSeriesFactory.js'
 import Module from './weblibdedx.js'
 
-
 /**
  * @typedef {LibdedxEntity}
  * @property {number} id - the libdedx ID assigned to the entity
  * @property {string} name - name of the entity read from libdedx
- */
-
-/**
- * @typedef {DataSeries}
- * @property {number[]} x - values to place on the X axis
- * @property {number[]} y - values to place on the Y axis
+ * 
+ * @typedef {PlotDataSeries}
+ * @property {number[]} energies - values of energy
+ * @property {number[]} powers - values of power
+ * 
+ * @typedef {CalculatorDataSeries}
+ * @property {number[]} energies - values of energy
+ * @property {number[]} powers - values of power
+ * @property {number[]} csda - values of CSDA range
```

Comment:
Same as above, combine `stopping` with `power`, otherwise name is too misleading.

Reply:
Also, mention `range`, as `range` has more meaning than `csda` (which itself is one of many methods to calculate range)